docs/reference/rollup/migrating-to-downsampling.asciidoc

```diff
@@ -0,0 +1,119 @@
+[role="xpack"]
+[[rollup-migrating-to-downsampling]]
+=== {rollup-cap} Migrating to downsampling
+++++
+<titleabbrev>Migrating to downsampling</titleabbrev>
+++++
+
+Rollup and downsampling are two different features that allow historical metrics to be rolled up.
+From a high level rollup offers more functionality compared to downsampling, but downsampling is a more robust and
+easier feature to downsample metrics.
+
+The following features are missing from downsampling:
+
+* Downsampling can only rollup metrics by the `_tsid` (combination of all dimensions) and `@timestamp` field. Rollup allows
+  to group by other fields. For example `@timestamp` and hostname or any other combination of fields.
+* Downsampling doesn't support calendar time intervals. Only fixed intervals are supported.
+* Downsampling can only downsample time series data streams, which means a data source needs to be metrics, with
+  configured dimension, metric fields and a `@timestamp` field. The rollup feature can rollup any index with a timestamp field.
+* Downsampling has less flexible scheduling control and it isn't possible to rollup data after short intervals. This
+  is because downsampling is tied to life cycle management. Essentially only after an index has been rolled
+  over it can be downsampled. This is because downsampling can only occur when a backing index is marked read only.
+
+The following aspects of downsampling are easier or more robust:
```

Comment:
I agree, I'd also want a lot more concrete migration instructions versus caveats and feature mismatches. Otherwise, I'd fear the user doesn't have strong motivation to migrate, or sufficiently clear guidance on _how to_ migrate.